source/tests/jax/test_make_hessian_model.py

```
@@ -0,0 +1,171 @@
+        ana_hess = ret_dict0["energy_derv_r_derv_r"]
+
+        # compute finite difference
+        fnt_hess = []
+        for ii in range(nf):
+
+            def np_infer(
+                xx,
+            ):
+                ret = self.model_valu(
+                    to_jax_array(xx)[None, ...],
+                    atype[ii][None, ...],
```

Ruff (0.8.2)

Function definition does not bind loop variable `ii`

(B023)